### ISSUE TITLE
Portfolio project code review

### ISSUE THREAD

**Issue by gsmalik030** [TamaraDimov/Portfolio-project#12]
I think you should delete code that you not using from your CSS on line 1 and not comment because it looks unprofessional.
I think all your sections should have a title or aria-labelledby attribute.
Some of your sections don't have a heading.
I think you should use letter spacing.
